src/Features/Core/Portable/UseIsNullCheck/AbstractUseIsNullDiagnosticAnalyzer.cs

```diff
@@ -144,7 +144,8 @@ private void AnalyzeSyntax(SyntaxNodeAnalysisContext context, IMethodSymbol refe
             var severity = option.Notification.Value;
             context.ReportDiagnostic(
                 Diagnostic.Create(
-                    GetDescriptorWithSeverity(severity), nameNode.GetLocation(),
+                    Descriptor, nameNode.GetLocation(),
```

Comment:
idk what style the parameters are using here, but it's bad.

Reply:
I agree, but I didn't change it in this pull request.